cmd/gangway/handlers.go

```diff
@@ -222,7 +241,11 @@ func callbackHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func commandlineHandler(w http.ResponseWriter, r *http.Request) {
+	info := generateInfo(w, r)
```

Comment:
We should handle the case where info == nil. Otherwise, we'll write to the ResponseWriter twice.